### PR TITLE
feature: ssl_certby_lua* support for stream_lua.

### PR DESCRIPTION
**Pull request by dndx**
This commit adds the support of ssl_certby_lua_* phases for stream_lua. Thanks Kong Inc. for sponsoring the development of this feature.

This PR was partially based on work performed by @james-callahan in https://github.com/openresty/meta-lua-nginx-module/pull/45, with modifications.

### CODE REVIEW

File: src/subsys/ngx_subsys_lua_ssl_certby.c.tt2 [hunk 1]
```diff
@@ -248,37 +258,67 @@ ngx_http_lua_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
...
 #else
 
+[% IF http_subsys %]
     fc->log->file = clcf->error_log->file;
 
     if (!(fc->log->log_level & NGX_LOG_DEBUG_CONNECTION)) {
         fc->log->log_level = clcf->error_log->log_level;
     }
+[% ELSIF stream_subsys %]
+#   error "stream ssl_cery_by_lua only supports nginx >= 1.13.0"
```
Comment (thibaultcha) [inline src/subsys/ngx_subsys_lua_ssl_certby.c.tt2]: typo: "cery" -> "cert"

File: src/subsys/ngx_subsys_lua_ssl_certby.c.tt2 [hunk 2]
```diff
@@ -248,37 +258,67 @@ ngx_http_lua_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
...
+[% ELSIF stream_subsys %]
+    ngx_set_connection_log(fc, cscf->error_log);
+[% END %]
 
 #   else
 
+[% IF http_subsys %]
     ngx_http_set_connection_log(fc, clcf->error_log);
+[% ELSIF stream_subsys %]
+#   error "stream ssl_cery_by_lua only supports nginx >= 1.13.0"
```
Comment (thibaultcha) [inline src/subsys/ngx_subsys_lua_ssl_certby.c.tt2]: ditto (typo)

**Comment by james-callahan** [comment]
Typo in commit message and PR title: "fearture" vs "feature"

**Comment by dndx** [comment]
@james-callahan Good catch. Commit message will be fixed when merging.

**Pull request merged**